core/src/main/java/org/stellar/anchor/event/KafkaEventService.java

```diff
@@ -43,6 +44,9 @@ public void publish(AnchorEvent event) {
       ProducerRecord<String, AnchorEvent> record = new ProducerRecord<>(topic, event);
       record.headers().add(new RecordHeader("type", event.getType().getBytes()));
       producer.send(record);
+      Metrics.counter(
```

Comment:
👍 It is awesome to use a vendor neutral interface! (`io.micrometer.core.instrument.Metrics`)